api/migrations/versions/2025_08_04_0001-add_account_email_lower_idx_func.py

```diff
@@ -0,0 +1,30 @@
+"""add_account_email_lower_idx_func
+
+Revision ID: add_account_email_lower_idx_func
+Revises: 532b3f888abf
+Create Date: 2023-08-15 00:01:00.000000
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'add_account_email_lower_idx_func'
+down_revision = '532b3f888abf'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # 为accounts表的email字段创建函数索引，使用lower函数
+    op.execute(
+        "CREATE INDEX IF NOT EXISTS account_email_lower_idx ON accounts (lower(email));"
```

Comment:
Since there were no limitations on case sensitivity before, there may be two different records with the same email in the historical data.